packages/sheet-permission-ui/src/views/error-msg-dialog/index.module.less

```diff
@@ -0,0 +1,27 @@
+.sheet-permission-alert-dialog {
+    padding: var(--padding-xl) 24px;
+
+    &-title {
+        font-size: var(--font-size-lg);
+        font-weight: 500;
+        margin-bottom: var(--margin-base);
+    }
+}
+
+.sheet-permission-alert-dialog-button {
+    margin-top: var(--margin-lg);
+    display: flex;
+    justify-content: flex-end;
+}
+
+:global(.sheet-permission-user-dialog .univer-dialog-close) {
```

Comment:
Can we use standard component here. I prefer no custom components.

Reply:
Because the container has been adjusted in the design draft, but the way we use it is to pass header, children, and footer; there is no place to configure the modified container，so solve it like this for now

Reply:
@jikkai Need an error dialog component here. Maybe.

Reply:
@ybzky Please add a todo note here. We should create a reusable component.